Documentation/gitattributes.txt

```diff
@@ -146,6 +146,33 @@ Unspecified::
 Any other value causes Git to act as if `text` has been left
 unspecified.
 
+`encoding`
+^^^^^^^^^^
+
+By default Git assumes UTF-8 encoding for text files.  The `encoding`
```

Comment:
Minor nagging about typography: The documentation doesn’t seem to use [English spacing](https://en.wikipedia.org/wiki/History_of_sentence_spacing#French_and_English_spacing) (two spaces between sentences) consistently.

While [many consider double sentence spacing outdated](https://en.wikipedia.org/wiki/History_of_sentence_spacing#Movement_to_single_sentence_spacing), I don’t know what the standard within Git is. This seems to be inconsistent right now, but maybe this should be checked. In case of doubt, I’d suggest modern single sentence spacing.

Reply:
I think @gitster prefers that. I dunno.